diff --git a/dist/plugin.js b/dist/plugin.js
index 236190fa9e545329f2939067a8b7c4e2ad82c67a..45a9f26e836beab33d5d7ff24f32d21aa9434761 100644
--- a/dist/plugin.js
+++ b/dist/plugin.js
@@ -30,7 +30,7 @@ const glob = __importStar(require("glob"));
 const helpers_1 = require("./helpers");
 function findDeployScripts(hre) {
     const workDir = hre.config.paths.root;
-    const deployScriptsDir = path.join(workDir, 'deploy');
+    const deployScriptsDir = path.join(workDir, 'contracts/aa/deploy');
     if (!(0, fs_1.existsSync)(deployScriptsDir)) {
         throw (0, helpers_1.pluginError)('No deploy folder was found');
     }
diff --git a/src/plugin.ts b/src/plugin.ts
index eef53173aafa41ba34deaf3cbe631e9d66354f7e..0903638fe0d22db3f391c33ced611d3489598fda 100644
--- a/src/plugin.ts
+++ b/src/plugin.ts
@@ -7,7 +7,7 @@ import { pluginError } from './helpers';
 
 export function findDeployScripts(hre: HardhatRuntimeEnvironment): string[] {
     const workDir = hre.config.paths.root;
-    const deployScriptsDir = path.join(workDir, 'deploy');
+    const deployScriptsDir = path.join(workDir, 'contracts/aa/deploy');
 
     if (!existsSync(deployScriptsDir)) {
         throw pluginError('No deploy folder was found');